src/ReactTerminal.js

```diff
@@ -122,6 +123,7 @@ class Terminal extends Component {
 };
 
 Terminal.propTypes = {
+  terminalId: PropTypes.number,
```

Comment:
Should be of type `string`: `PropTypes.string`.

Reply:
Good catch, should be fixed now :dancer: